src/akamai/js/akamaiHD_flash.js

```diff
@@ -201,8 +211,29 @@ require("../../../html5-common/js/utils/constants.js");
      * @returns {boolean} True or false indicating success
      */
     this.setVideoUrl = function(url, encoding) {
-      /* Should be set to false if stream URL changes */
-      loaded = false;
+      newController=this.controller;
```

Comment:
It's good that this variable no longer global - but why do we set it in setVideoUrl?